src/containers/Configure/Configure.js

```diff
@@ -51,74 +51,80 @@ const Configure = ({ userQuote, allUserQuotes, loadSolarData, quote }) => {
               <h3>Configure your array:</h3>
               {error && <p className="error">{error}</p>}
               <div className="configure-form-item">
-                <label>System Size (kW): (1-10)</label>
-                <input
-                  type="number"
-                  placeholder="4"
-                  value={systemSize}
-                  required
-                  onChange={e => updateSystemSize(parseFloat(e.target.value))}
-                />
+                <label>System Size (kW) (1-10):
```

Comment:
I like the way these input fields are nested in there labels.  I tried to implement that in the historical component but had to revert it due to complications with the way the styling was set up on that one

Reply:
I can see how that would affect styling